includes/theme-compat/header.php

```diff
@@ -0,0 +1,52 @@
+<?php
+//require_once (dirname(__FILE__).'/config/config.php');
+require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/config.php');
+require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/functions.php');
+require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/db-connect.php');
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jobs</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+
+
+    <?php echo "<link rel=\"stylesheet\" href=\"$HTTP_DOMAIN/assets/css/master.css\">"; ?>
```

Comment:
I already mentioned it in another comment, but let's use the `BASE_URL` define instead of `$HTTP_DOMAIN`